trashtalk/apps/cleanups/models.py

```diff
@@ -114,3 +114,45 @@ def has_state_code(self):
 
     def is_intersection(self):
         return self.category == 'intersection'
+
+# pylint: disable=missing-docstring
+class ToolCategory(models.Model):
```

Comment:
I see what you're trying to do but I don't think this is the best way to implement this. It's a lot simpler than 3 models.

Reply:
I would argue that three models is the way to go. However, I would change the structure to look closer to ...

Reply:
@Protosac are you proposing that these should just be field choices? If so, I'd assert that there are a few different advantages to modeling this data using these models instead of storing them as field choices in code:

* Storing tools in the DB is more scale-able and easier for non-devs to manage through the Django Admin
* Updates to tools wouldn't require a server update. If we codify these options in Django files, wouldn't we have to do a server re-deploy whenever this list changes? Having tools in tables means we can add/remove/activate the tools in Django admin
* I'm not sure how we'd organize these options on the frontend in sane way without breaking them up into categories. I don't think categorization is possible using field options?

Reply:
@thinhngo If we replaced Tool Categories as a separate model with a property/field of each Tool, then we would either: 
```For category in categories: Select * From Tool Where Category Equals [category]``` 
or 
``` Select * From Tool``` and then sort the categories in the view function. 

I would also like an Order model for two reasons:
 1. All of the tools and their quantities are collected into one model linked with a cleanup & host. 
      a. By collecting them together, it is easier to create a single page to manipulate the whole order
      b. Keeping all of the information in one page makes it easier to protect with an ```@host_required``` decorator
 2. Storing a summary of an order would allow the order to be saved to a user account and duplicated for other cleanups.

Reply:
1. We can't code for users we don't have. We're not worried about scale at this stage.
2. The process for cleanup hosts is fairly experimental at this stage as far as how this will be managed.
3. A choice field offers us exactly what we need and is an easier approach to evolving the tool request: flexibility and simplicity.

We'll discuss a bit more tonight.